mica/report/report.py

```diff
@@ -216,7 +217,7 @@ def get_obs_trak_stats(obsid):
         return Table(guis)
 
 
-def get_obs_temps(obsid, outdir):
+def get_obs_temps(obsid):
```

Comment:
This is a real API change to remove an unused argument.

Reply:
👍🏼